### PR TITLE
Change fake data column names and drop one

### PR DESCRIPTION
**Pull request by nacnudus**
NHS column names now to be:

```
'Traced_NHSNUMBER',
'DateOfBirth',
'PatientFirstName',
'PatientOtherName',
'PatientSurname',
'PatientAddress_Line1',
'PatientAddress_Line2',
'PatientAddress_Line3',
'PatientAddress_Line4',
'PatientAddress_Line5',
'PatientAddress_PostCode',
'GPPractice_Code',
'Practice_NAME',
'contact_telephone',
'mobile',
'landline',
'oslaua',
'ccg',
'Flag_Chemo/Radiotherapy',
'Flag_Respiratory',
'Flag_HeamatologicalCancers',
'Flag_PregnantWithCongentialHeartDefect',
'Flag_Transplant',
'Flag_RareDiseases',
'Gender',
'Flag_PDSInformallyDeceased',
'oscty'
```

## AQA Checklist:

- [ ] The column names of the `data/fake-data/nhs.csv` are as above, and are correctly aligned with the data.
- [ ] Ditto for the first set of columns of the `data/fake-data/master.csv`

### CODE REVIEW

File: src/make_data/fake-data.R [hunk 1]
```diff
@@ -334,12 +332,16 @@ nhs_list <- sample_frac(master, nhs_perc + nhs_and_web_perc)
 web_list <- sample_frac(master, web_perc + nhs_and_web_perc)
 ivr_list <- sample_frac(nhs_list, ivr_perc)
 
-nhs_list <- select_at(nhs_list, nhs_column_names)
-web_list <- select_at(web_list, web_column_names)
+# nhs_list <- select_at(nhs_list, nhs_column_names)
+# web_list <- select_at(web_list, web_column_na
+
+mes)
```
Comment (avisionh) [inline src/make_data/fake-data.R:338]: Looks like one of those times when you want to paragraph when writing a reply/document but do so on the wrong window?

```suggestion
# web_list <- select_at(web_list, web_column_names)
```

File: src/make_data/fake-data.R [hunk 2]
```diff
@@ -334,12 +332,16 @@ nhs_list <- sample_frac(master, nhs_perc + nhs_and_web_perc)
...
 ivr_list <- select_at(ivr_list, ivr_column_names)
 
+transactions <- t$events
+names(transactions) <- t$session_id
+transactions <-
```
Comment (avisionh) [inline src/make_data/fake-data.R:343]: Was something meant to go to the right of the assignment operator?

File: src/make_data/fake-data.R [hunk 3]
```diff
@@ -154,36 +154,35 @@ master <-
...
+    PatientAddress_Line2 = map_chr(addresses, ~ .x$city()),
+    PatientAddress_Line3 = map_chr(addresses, ~ .x$street_name()),
+    PatientAddress_Line4 = map_chr(places$name_1, ~ ifelse(is.null(.x), NA_character_, .x)),
+    PatientAddress_Line5 = map_chr(places$county_unitary, ~ ifelse(is.null(.x), NA_character_, .x)),
+    PatientAddress_Postcode = postcodes,
+    GPPractice_Code = nhs_gp_practice_codes$code,
+    Practice_NAME = nhs_gp_practice_codes$name,
     contact_telephone = nhs_gp_practice_codes$telephone,
     mobile = ch_phone_number(n, locale = "en_GB"),
     patient_landline = ch_phone_number(n, locale = "en_GB"),
```
Comment (avisionh) [inline src/make_data/fake-data.R:172]: Should this change to just `landline` as per requirements?

```suggestion
    landline = ch_phone_number(n, locale = "en_GB"),
```

File: src/make_data/fake-data.R [hunk 4]
```diff
@@ -249,33 +248,32 @@ write.csv(x = master, file = here("data/fake-data/master.csv"), quote = TRUE, ro
...
+    "PatientAddress_Line2",
+    "PatientAddress_Line3",
+    "PatientAddress_Line4",
+    "PatientAddress_Line5",
+    "PatientAddress_Postcode",
+    "GPPractice_Code",
+    "Practice_NAME",
     "contact_telephone",
     "mobile",
     "patient_landline",
```
Comment (avisionh) [inline src/make_data/fake-data.R:266]: Should this change to just `landline` as per requirements?

```suggestion
    landline = ch_phone_number(n, locale = "en_GB"),
```

File: src/make_data/fake-data.R [hunk 5]
```diff
@@ -334,12 +332,16 @@ nhs_list <- sample_frac(master, nhs_perc + nhs_and_web_perc)
 web_list <- sample_frac(master, web_perc + nhs_and_web_perc)
 ivr_list <- sample_frac(nhs_list, ivr_perc)
 
-nhs_list <- select_at(nhs_list, nhs_column_names)
-web_list <- select_at(web_list, web_column_names)
+# nhs_list <- select_at(nhs_list, nhs_column_names)
+# web_list <- select_at(web_list, web_column_na
```
Comment (avisionh) [inline src/make_data/fake-data.R:336]: Are we deliberately commenting these out? If so, we aren't able to apply the new column name, `Flag_HaemotologicalCancers` to the current name, `Flag_HaemotologicalCancers`, as per the request.

```suggestion
nhs_list <- select_at(nhs_list, nhs_column_names)
web_list <- select_at(web_list, web_column_na
```

File: src/make_data/fake-data.R (hunk 1 above)
Comment (nacnudus) [reply src/make_data/fake-data.R:338]: :grimacing: and then don't re-run the code :face_with_head_bandage:

File: src/make_data/fake-data.R [hunk 6]
```diff
@@ -381,7 +383,7 @@ ivr_list %>%
 ivr_list %>%
   anti_join(web_list, by = c("ivr_nhs_number" = "nhs_number")) %>%
   nrow()
-
+as(transactions, "transactions")
```
Comment (nacnudus) [inline src/make_data/fake-data.R:386]: Ditto.

File: src/make_data/fake-data.R (hunk 5 above)
Comment (nacnudus) [reply src/make_data/fake-data.R:336]: :man_facepalming:

File: src/make_data/fake-data.R (hunk 4 above)
Comment (nacnudus) [reply src/make_data/fake-data.R:266]: Good spot, thanks.

File: src/make_data/fake-data.R (hunk 3 above)
Comment (nacnudus) [reply src/make_data/fake-data.R:172]: Good spot, thanks.

File: src/make_data/fake-data.R (hunk 2 above)
Comment (avisionh) [reply src/make_data/fake-data.R]: Maybe it was me in a previous branch! 😅

**Pull request merged**